test/metric_value_benchmark/agent_configs/prometheus_config.json

```diff
@@ -20,10 +20,10 @@
               "label_matcher": "^yes$",
               "dimensions": [
                 [
-                  "prom_metric_type"
+                  "prom_type"
                 ],
                 [
```

Comment:
**Nit**: If comment out summary, should it be the same for dimension "quantile"? Yes, it will work but I assume it would be better to come along when we officially support summary with CCWA

Reply:
Agree.. I thought Id keep the changes minimal to revert later on. Since the metrics to measure itself doesnt have summary, the quantile or the extra dimension below wont be used.
If itll be cleaner to comment out in other places as well, I can update it.